oarepo_runtime/services/files/components.py

```diff
@@ -0,0 +1,59 @@
+import mimetypes
+import os
+from invenio_records_resources.services.files.components import FileServiceComponent
+
+class FileTypeException(Exception):
```

Comment:
This might be a subclass of some `ValidationError`? Needs to be serializable to response (not produce 500 error)